src/runner/restart_elevated.cpp

```diff
@@ -7,15 +7,17 @@ enum State
 {
     None,
     RestartAsElevated,
+    RestartAsElevatedOpenSettings,

```

Comment:
SG: replace `RestartAsElevatedOpenSettings` by `OpenSettings` and use powers of two as integral values for enum. It is better to have independent options.